docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,437 @@
+# Basic Usage
+
+Dojo provides a reactive architecture concerned with constantly modifying and rendering the current state of an application. In simple systems this can happen at a local level and widgets can maintain their own state. However, as a system becomes more complex the need to better compartmentalize and encapsulate data and create a clean separation of concerns quickly grows.
+
+Stores provide a clean interface for storing, modifying, and retrieving data from a global object through unidirectional data flow. Stores include support for common patterns such as asynchronous data retrieval, middleware, and undo. Stores and their patterns allow widgets to focus on their primary role of providing a visual representation of information and listening for user interactions.
+
+## The Store
+
+The store holds the global, atomic state for the entire application. The store gets created when the application gets created and defined in the `Registry` with an injector.
+
+> main.ts
+
+```ts
+import { registerStoreInjector } from '@dojo/framework/store/StoreInjector';
+import Store from '@dojo/framework/stores/Store';
+import { State } from './interfaces';
+
+const store = new Store<State>();
+const registry = registerStoreInjector(store);
+```
+
+``State`defines the structure of the global store using an interface. Everything inside`State` should be serializable, i.e. convertable to/from JSON, as this improves performance by making it easier for the Dojo virtual DOM system to determine when changes to the data occur.
+
+> interfaces.d.ts
+
+```ts
+interface User {
+	id: string;
+	name: string;
+}
+
+export interface State {
+	auth: {
+		token: string;
+	};
+	users: {
+		current: User;
+		list: User[];
+	};
+}
+```
+
+The above is a simple example that defines the structure for the store used in the rest of the examples in this guide.
+
+## Updating stores
+
+There are three core concepts when working with Dojo stores.
+
+-   **Operations** - instructions to manipulate the state held by the store
+-   **Commands** - simple functions that perform business logic and return operations
+-   **Processes** - execute a group of commands and represent application behavior
+
+### Commands & Operations
+
+To modify a store, when executing a process, an operation function gets invoked. The operation function gets returned from a command. Each command is passed a `CommandRequest` which provides `path` and `at` functions to generate `Path`s in a type-safe way, a `get` function for access to the store's state, a `payload` object for the argument that the process executor was called with.
```

Comment:
Should this sentence say "command function" instead of "operation function"? And in the next sentence, "The operation function gets returned from a command", is that right? Commands return an array of operations (or a promise or nothing), not a function.